benchmarks/benchmark_messages_proto2.proto

```diff
@@ -1,11 +1,14 @@
+// Benchmark messages for proto2.
+
 syntax = "proto2";
 
-package benchmarks;
+package benchmarks.p2;
```

Comment:
It's not a common idiom to abbreviate proto2 as p2 (or proto3 as p3). Just use "benchmarks.proto2"?